styles/sass/tables.scss

```diff
@@ -7,7 +7,7 @@
 // BASE TABLES
 // -----------------
 
-table {
+.table {
```

Comment:
I wanted to use a table to show authors and roles, but it had lots of default styles that I would have to overwrite. Instead I changed those rules so they only apply to elements with a class name instead of being global. I also did the same in the UI framework and updated current markup to rely on these classes instead of global styles.

Yep, the rules should be merged, I'll update the PR.